.vscode/settings.json

```diff
@@ -34,5 +34,8 @@
   // Screw redhat and their telemetry
   "redhat.telemetry.enabled": false,
   // Always use single quotes in yml files
-  "yaml.format.singleQuote": true
+  "yaml.format.singleQuote": true,
+  "[solidity]": {
+    "editor.defaultFormatter": "NomicFoundation.hardhat-solidity"
+  }
```

Comment:
@cha0sg0d do we know if this formats things the same as prettier (since we run prettier on solidity in CI)

Reply:
How do we test? By just running on CI and seeing if tests pass? or running `Format Document` in VSCode vs prettier from command line?

Reply:
I think if it doesn't produce a formatting commit on this PR then it is the same 🙃